code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -739,6 +739,14 @@
 	burntime = 1
 	var/affects_servants = FALSE
 	var/stat_affected = CONSCIOUS
+	var/resist_string = "blinding white" //string for when a null rod blocks its effects, "glows [resist_string]"
```

Comment:
how about instead of "glows [resist_string]"  we just have "[resist_string]" and add "glows " to the start of every resist_string.  That way we could have "shimmers deeply" or something as a string and have it make sense.